plugins/statistics.koplugin/main.lua

```diff
@@ -2643,6 +2691,15 @@ end
 
 
 function ReaderStatistics:onPosUpdate(pos, pageno)
+    if self.use_pagemap_for_stats then
+        local __, page_sequence_number = self.ui.pagemap:getCurrentPageLabel()
```

Comment:
Forgot to switch that one to `select(2...)` ?
I think you can call it `local curr_page` - or `pageidx` (to keep the notion of sequence, and distinguish it from `pageno` below if it is indeed different).